src/Kmd.Logic.Gateway.Automation/Models/ApiValidationResult.cs

```diff
@@ -19,15 +19,18 @@ public class ApiValidationResult : ItemValidationResultBase
 
         public override string ToString()
         {
-            var api = $"API: '{this.Name}' - {this.Path}/{this.Version}";
-            var result = this.ToString(api);
-
-            result += this.ApiId.HasValue ? $"* Api ID: {this.ApiId.Value}" : string.Empty;
-            result += this.ApiVersionSetId.HasValue ? $"* API version set: {this.ApiVersionSetId.Value}" : string.Empty;
+            var result = $"* API Name: {this.Name}\n";
+            result += $"* Path: {this.Path}\n";
+            result += $"* Version: {this.Version}\n";
```

Comment:
Use string builder, all places where we have more string concatenation

Reply:
fixed